core/src/main/java/org/ldaptive/AbstractMessage.java

```diff
@@ -220,6 +223,11 @@ public ControlParser()
       parser.registerHandler(CRITICAL_PATH, (p, e) -> critical = BooleanType.decode(e));
       parser.registerHandler(OID_PATH, (p, e) -> oid = OctetStringType.decode(e));
       parser.registerHandler(VALUE_PATH, (p, e) -> value = e.slice());
+      parser.registerHandler(ALT_VALUE_PATH, (p, e) -> {
+        if (value == null) {
+          value = e.slice();
+        }
```

Comment:
```suggestion
        if (value != null) {
          throw new IllegalArgumentException("Multiple control values found");
        }
        value = e.slice();
```

Reply:
Maybe a log message is just as useful here, but some sort of indicator that an invalid sequence was received seems merited.

Reply:
Actually, this is fine the way it is. It lines up with the way other parsers work.